lib/py/src/TMultiplexedProcessor.py

```diff
@@ -17,9 +17,9 @@
 # under the License.
 #
 
-from thrift.Thrift import TProcessor, TMessageType
-from thrift.protocol import TProtocolDecorator, TMultiplexedProtocol
-from thrift.protocol.TProtocol import TProtocolException
+from .thrift.Thrift import TProcessor, TMessageType
```

Comment:
Was this change needed? I'm not a fan of relative imports because.

Reply:
@Fokko do you want to help with this as well? we can create separate PR to track it. for now i want to keep the change minimal. this change was introduced by 2to3 script automatically.